src/Main.jsx

```diff
@@ -26,7 +28,7 @@ const formatPercent = new Intl.NumberFormat('en-US', {
 })
 
 const Main = (props) => {
-  document.title = `Dai Stats - ${formatAmount.format(props.debt)}`
+  document.title = `Dai Stats`
```

Comment:
I removed this because this is the value used when metamask asks for approval, and the total DAI balance in there looked like it would scare away users.